src/Uno.UWP/Media/Playback/MediaPlayer.cs

```diff
@@ -89,12 +90,19 @@ public double Volume
 
 		#endregion
 
+#if __WASM__
+		public MediaPlayer(IHtmlMediaPlayer player)
```

Comment:
Is there a way you could avoid this constructor? It is not part of the UWP/WinUI contract - and the parameterless constructor should remain there - as it is `public` - https://learn.microsoft.com/en-us/uwp/api/windows.media.playback.mediaplayer.-ctor?view=winrt-22621